pystiche_papers/ulyanov_et_al_2016/_nst.py

```diff
@@ -23,6 +24,26 @@
 __all__ = ["training", "stylization"]
 
 
+class OptionalResizeCenterCropToMultiple(nn.Module):
+    def __init__(self, multiple: int = 64):
+        super().__init__()
+        self.multiple = multiple
+
+    def forward(self, input_image: torch.Tensor) -> torch.Tensor:
+        old_height, old_width = input_image.shape[-2:]
+        if old_height % 64 == 0 and old_width % 64 == 0:
+            return input_image
+
+        min_length = min([old_height, old_width])
+        new_length = math.ceil(min_length / self.multiple) * self.multiple
+        transforms_: List[nn.Module] = []
+        transforms_.append(transforms.Resize(new_length))
+        transforms_.append(transforms.CenterCrop(new_length))
+        transform = nn.Sequential(*transforms_)
+
+        return cast(torch.Tensor, transform(input_image))
```

Comment:
Instead of creating a transform object here and afterwards only calling it once, you can use the functional API directly:

```suggestion
        # move this to the other imports
        from torchvision.transforms import functional as F
        
        output_image = F.resize(input_image, new_length)
        output_image = F.center_crop(output_image, new_length)
        return output_image
```